core/src/main/thrift/compaction-coordinator.thrift

```diff
@@ -43,9 +43,9 @@ enum TCompactionState {
 struct TCompactionStatusUpdate {
   1:TCompactionState state
   2:string message
-  3:i64 bytesToBeCompacted
-  4:i64 bytesRead
-  5:i64 bytesWritten
+  3:i64 entriesToBeCompacted
+  4:i64 entriesRead
+  5:i64 entriesWritten
 }
```

Comment:
I missed why these were changed. I did a rebase on your branch and this broke some stuff on my branch. I think its OK just wondering. Are these values not bytes?

Reply:
Yes, I'm pretty sure it's not bytes and it *is* entries. The genesis for those metrics is [here](https://github.com/dlmarion/accumulo/blob/expose-external-compaction-progress-for-monitor/server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java#L684). If you dig into that you get to:

FileCompactor.getRunningCompactions() -> new CompactionInfo(FileCompactor)